13-microbiome-exploration.Rmd

```diff
@@ -31,71 +31,133 @@ Investigating the prevalence of microbes allows you either to focus on changes,
 which pertain to most of the samples, or to focus on less often found microbes,
 which are nonetheless abundantly found in a small number of samples.
 
-Population prevalence (frequency) at a 1% relative abundance threshold:
+On the raw data, the population prevalence (frequency) at a 1% relative
+abundance threshold (`detection = 1/100` and `as_relative = TRUE`), can look
+like this. The low prevalence in this examples can be explained by rather
+different sample types as well as the in depth nature of the data.
 
 ```{r exploration-prevalence}
 head(getPrevalence(se, detection = 1/100, sort = TRUE, as_relative = TRUE))
 ```
 
-Population prevalence (frequency) at the absolute abundance threshold at read count 1:
+The `detection` and `as_relative` can also be used to access, how many samples
+do pass a threshold for raw counts. Here the population prevalence (frequency) 
+at the absolute abundance threshold (`as_relative = FALSE`) at read count 1
+(`detection = TRUE`) is accessed.
```

Comment:
This is good and clear text but makes me think where do we draw the line between education book vs. technical manual.. the former emphasizes fluent readability and perhaps shorter expression, the latter emphasizes completeness and exactness. Sometimes it is difficult to achieve both simultaneously. So it is also a general question of the writing style.